docs/index.md

```diff
@@ -30,15 +30,15 @@ our images start at 400MB in size (3x lighter than 'Raspbian Lite'). With featur
 
 ## Getting started
 
-If you are **new to DietPi**, the [_Install_](user-guide_install) provides a tutorial for everything you need to get DietPi running. 
+If you are **new to DietPi**, the [_Install_](user-guide_install_sbc) provides a tutorial for everything you need to get DietPi running. 
 
 If you **need help**, and have read through [_Getting Started_](user-guide_overview), check out [_Troubleshooting_](https://dietpi.com/phpbb/viewforum.php?f=11).
 
 For more details visit the [_DietPi Community Forum_](https://dietpi.com/phpbb/viewforum.php?f=5).
 
 ## Easy to install popular software
 
-You can **quickly and easily install popular software** ! All of them are **READY TO RUN** and optimised for your system. The list of available apps is long , and for the full list checkout [_DietPi Optimised Software_](user-optimised-software) page. Here are few highlights:
+You can **quickly and easily install popular software** ! All of them are **READY TO RUN** and optimised for your system. The list of available apps is long , and for the full list checkout [_DietPi Optimised Software_](dietpi_optimised_software) page. Here are few highlights:
```

Comment:
As we said we want to keep the URLs short, still meaningful, what about `dietpi_software` here? Although it could be reasonable to keep that one free for a separate documentation about the `dietpi-software` UI, or to not be too close to that when using `dietpi-software` (dash instead of underscore).